### PR TITLE
Feature/drinking water service change

### PR DESCRIPTION
**Pull request by oftheheadland**
## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3308677

## Main Changes:
* Switch to new drinking water service that uses state and county codes from the NEPAssist county service.
* Withdrawers tab of Community Drinking Water tab now uses water_type_calc property to determine whether withdrawers are Surface Water or Ground Water.

## Steps To Test:
Note: new drinking water service is returning duplicates for some withdrawers where they will have the same pwid and data but the water_type_calc property is different. 

Example of the duplicate issue: 

Las, Vegas. First 2 items in service results https://ofmpub.epa.gov/apex/sfdw_rest/GetPWSWMHUC12FIPS/150100150604/32/003

We have contacted the service to make sure they are aware and HMW is able to handle the duplicate issue.

You may need to clear cache and refresh to fix CORS issues when comparing between localhost and the dev site. This is an old Chromium issue we have run into before and are looking into it in another ticket.

1. On Community, navigate to some test locations listed below, then go to the Withdrawers tab of the Drinking Water tab:
http://localhost:3000/community/las%20vegas/drinking-water 
http://localhost:3000/community/010900020303/drinking-water

2. Verify Withdrawers tab is now using the water_type_calc property instead of the gw_sw property from the drinking water service data:
- Compare these locations to the dev site and verify 'Groundwater' is now 'Ground Water' on the local branch. 
- Compare the number of withdrawers in each category between the dev and live site for Las, Vegas. The numbers should be different.
- Verify the sorts work on the Drinking Water tab

### CODE REVIEW

File: app/client/src/components/pages/Community/components/tabs/DrinkingWater.js [hunk 1]
```diff
@@ -390,7 +418,7 @@ function DrinkingWater({ esriModules, infoToggleChecked }: Props) {
     },
     {
       value: 'source',
-      label: 'Drinking Water Source',
+      label: 'Drinking Water Facility Source',
```
Comment (cschwinderg) [inline app/client/src/components/pages/Community/components/tabs/DrinkingWater.js:421]: Similar to above.

**Comment by cschwinderg** [comment]
I noticed another issue. If you got to [http://localhost:3000/community/auburn%20al/drinking-water](http://localhost:3000/community/auburn%20al/drinking-water), the counts next to the filter switches are off. Surface water says 2 but if you only have the surface water toggle on there are 3 items in the list. It looks like it is likely due to an additional water_type_code (Ground water under the influence of surface water).

**Comment by oftheheadland** [comment]
@cschwinderg regarding your last comment about Aubun, AL it looks like the duplicate items in the service is causing issues. There is a duplicate key error in the console and after toggling them a few times extra duplicates appear (there should be 4 items)
image

**Comment by oftheheadland** [comment]
I'll look into these thank you

**Comment by oftheheadland** [comment]
@cschwinderg the label issue should be resolved and I've added another property to the accordion keys to prevent duplicate issues like the ones in Ashburn AL

**Pull request merged**